packages/wordcount/src/index.js

```diff
@@ -0,0 +1,119 @@
+import _ from 'lodash';
```

Comment:
Should we import directly the dependencies we need: `import { extend } from 'lodash'`?
And maybe we can use https://github.com/lodash/babel-plugin-lodash to allo tree-shaking